Makefile

```diff
@@ -266,3 +271,13 @@ tools/bin/misspell:tools/check/go.mod
 tools/bin/ineffassign:tools/check/go.mod
 	cd tools/check; \
 	$(GO) build -o ../bin/ineffassign github.com/gordonklaus/ineffassign
+
+# Usage:
+#
+# 	$ make vectorized-bench VB_FILE=Time VB_FUNC=builtinCurrentDateSig
+vectorized-bench:
```

Comment:
PTAL @qw4990

Reply:
I think we don't need this since `go test -v -benchmem -bench=xxx -run=xxx -args xxx` is simple enough.
What are your opinions? @SunRunAway @Reminiscent

Reply:
Since the `-args` is not the standard go test argument, it's ok to me to leave them in `Makefile` for standardization.

Reply:
For a beginner like me, it's just was not very obvious how to test new vectorized functions.
I could not find any README or any other document explaining the proccess. Only by searching i found original PR/12153.

So i just decided to fix the way of testing in Makefile.
If you think it's not needed - it's ok, i can remove it.

Reply:
Okay, we can keep it.